<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreatorC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CodeEditor" name="plainTextEdit">
       <property name="font">
        <font>
         <family>Cascadia Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/add-document.png</normaloff>:/imgs/Recourses-Black/add-document.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/folder-open.png</normaloff>:/imgs/Recourses-Black/folder-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/undo.png</normaloff>:/imgs/Recourses-Black/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/redo.png</normaloff>:/imgs/Recourses-Black/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/copy-alt.png</normaloff>:/imgs/Recourses-Black/copy-alt.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/scissors.png</normaloff>:/imgs/Recourses-Black/scissors.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/paste.png</normaloff>:/imgs/Recourses-Black/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/diskette.png</normaloff>:/imgs/Recourses-Black/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/save-as.png</normaloff>:/imgs/Recourses-Black/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/imgs/Recourses-Black/shuttle.png</normaloff>:/imgs/Recourses-Black/shuttle.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>D:\Qt-projects\MinGW\CreatorC\codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
